src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -79,26 +65,6 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
-    @Test
-    public void execute_filteredList_success() {
-        showParticipantAtIndex(model, INDEX_FIRST_PARTICIPANT);
```

Comment:
I give up, pass for me 😢

Reply:
They all pass locally for me too. But not Github 😰